test/sanity/issue5730-add-node-in-permissions/test.py

```diff
@@ -0,0 +1,26 @@
+import time
+import os
+
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+
+chrome_options = Options()
+testdir = os.path.dirname(os.path.abspath(__file__))
+chrome_options.add_argument('nwapp=' + testdir)
+
+driver = webdriver.Chrome(executable_path=os.environ['CHROMEDRIVER'], chrome_options=chrome_options)
+driver.implicitly_wait(3)
+try:
+    print driver.current_url
+    time.sleep(1)
```

Comment:
There may be race condition here. Please remove time.sleep() and use loop and wait as we talked.